manifests/vanilla/validatingwebhook.yaml

```diff
@@ -116,8 +116,19 @@ spec:
         role: vsphere-csi-webhook
     spec:
       serviceAccountName: vsphere-csi-webhook
-      nodeSelector:
-        node-role.kubernetes.io/control-plane: ""
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+            - matchExpressions:
+              - key: node-role.kubernetes.io/control-plane
+                operator: Exists
+            - matchExpressions:
+              - key: node-role.kubernetes.io/controlplane
```

Comment:
Remove this selector expression. Looks like a typo

Reply:
There is a following comment on GitHub issue, so looks like we may need to keep this selector expression as well.

```
Some k8s distributions use "control-plane" other use "[controlplane](https://rke.docs.rancher.com/config-options/nodes#controlplane)".

The Azure CSI driver already uses the affinity stanza provided above.

https://github.com/kubernetes-sigs/azuredisk-csi-driver/blob/bc6ce67e47376ccea4c639dadc7fb5e3feb17ad5/deploy/v1.30.0/csi-snapshot-controller.yaml#L38

And there's a lot of other projects that use the affinity explicitly to avoid this "control-plane" vs "controlplane" issue.
```